run.py

```diff
@@ -34,8 +34,8 @@
 from enum import Enum
 from multiprocessing import Pool
 from typing import NamedTuple
-
-from vunit import VUnit, VUnitCLI  # type: ignore
+from vunit.ui import VUnit
+from vunit.vunit_cli import VUnitCLI
```

Comment:
```suggestion

from vunit.ui import VUnit # type: ignore
from vunit.vunit_cli import VUnitCLI # type: ignore
```

MyPy will complain if modules don't have stubs, hence the `# type: ignore`  to those particular lines. Also, if you can format with [black](https://pypi.org/project/black/) it helps a lot with consistency (the extra line before the VUnit imports) :)